source/client/service_impl.h

```diff
@@ -11,8 +11,8 @@
 
 #include <future>
 
-#include "common/common/logger.h"
-#include "common/common/thread.h"
+#include "common/common/thread.h" // XXX(oschaaf)
+#include "external/envoy/source/common/common/logger.h"
```

Comment:
Can you consistently place `external/envoy` in its own block?

Reply:
Done